src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -73,9 +73,6 @@
     <Compile Include="$(CommonTestPath)\System\ObjectCloner.cs" >
       <Link>Common\System\ObjectCloner.cs</Link>
     </Compile>
-    <Compile Include="$(CommonTestPath)\System\RuntimeDetection.cs" >
```

Comment:
[Github search](https://github.com/dotnet/corefx/search?utf8=%E2%9C%93&q=RuntimeDetection) says it's still being used for System.Runtime tests and System.Text.Encoding tests, but there's no reason why it needs to be. 

I'll make those tests use SkipOnTargetFramework and delete RuntimeDetection.cs